Answers.md

```diff
@@ -1,3 +1,22 @@
 ### Your responses to the short answer questions should be laid out here using markdown.
+1. Describe the differences between a Docker container and a virtual machine. What makes a container more aptly-suited for portability?  
+>A docker container is different from a virtual machine because it is leaner and specialized for a much smaller range of tasks. It is more aptly  
+>suited for portability because it takes up far fewer resources to run.
+
+
+2. Given the commend `docker run -p 49160:8080 -d <your_docker_username>/<your_docker_image_name>`, what does `49160:8080` specify?  
+>49160:8080 specificies that 49160 will be how someone can reach the image of the item that was hosted on localhost:8080 in your docker image
+
+3. What is the main purpose of using a "container orchestration platform" such as Kubernetes or Docker Swarm?  
+>  These tools limit the amount of containers and resources being used according to the demand
```

Comment:
Not exactly. Orchestration platforms allow developers to more easily provision and coordinate multiple Docker images that all need to work in concert. Platforms like Kubernetes and Docker Swarm have tools that make it easier to spin up, move, and dedicate system resources to each container.